concepts/ORM/model-settings.md

```diff
@@ -151,7 +151,7 @@ attributes: {
 }
 ```
 
-See [Attributes]().
+See [Attributes](http://sailsjs.org/#/documentation/concepts/ORM/Attributes.html).
```

Comment:
Yes I felt the same way, but I was copying existing patterns because I don't know what a relative path would have been. More accurately, all the links are useless in markdown. I sort of wished the docs were more github friendly.